samples/cpp/benchmark_app/main.cpp

```diff
@@ -193,6 +193,21 @@ int main(int argc, char* argv[]) {
             slog::info << "GPU extensions is loaded " << ext << slog::endl;
         }
 
+        if (FLAGS_hint.empty()) {
+            for (auto& device : devices) {
+                std::vector<std::string> supported_config_keys =
+                    core.get_metric(device, METRIC_KEY(SUPPORTED_CONFIG_KEYS));
+                if (std::find(supported_config_keys.begin(),
+                              supported_config_keys.end(),
+                              CONFIG_KEY(PERFORMANCE_HINT)) != supported_config_keys.end()) {
+                    slog::warn << "-hint default value is determined as" << CONFIG_VALUE(THROUGHPUT)
+                               << " automatically for " << device
+                               << " device. For more detailed information look at README." << slog::endl;
+                    FLAGS_hint = CONFIG_VALUE(THROUGHPUT);
```

Comment:
Have you checked that this works? e.g. no 
the line 204 below checks for "throughput" (lower case), while CONFIG_VALUE(THROUGHPUT) sets the "THROUGHPUT"...

the check in the line 204 (https://github.com/openvinotoolkit/openvino/pull/9107/files#diff-e7a536b4318b2270a4ee67f723b8ff57e9a5aed0928c11fa46e5a0b02db6300fL204) should do smth like std::toupper first